base/stat.jl

```diff
@@ -351,22 +383,29 @@ ispath(path::AbstractString) = ispath(String(path))
 
 """
     isfifo(path) -> Bool
+    isfifo(path_elements...) -> Bool
+    isfifo(stat_struct) -> Bool
 
-Return `true` if `path` is a FIFO, `false` otherwise.
+Return `true` if the file at `path` is FIFO, `false` otherwise.
+Or the equivalent as indicated by the file descriptor `stat_struct`.
```

Comment:
```suggestion
Return `true` if the file at `path` or file descriptor `stat_struct` is FIFO, `false` otherwise.
```
I think this is more clear, do you agree? I am happy to use your language if you prefer.

Reply:
No that looks good to me.

Reply:
Would you please make this change in all the other places where it applies. Any docstring where
- There's a description for path and then an "or equivalent" clause or similar and
- It would be more clear to use the "file at `path` or file descriptor `stat_struct`" language

Reply:
Done. It was a bit more ankways in the "points at" cases, but I think I found a good compromise.